.github/workflows/build.yml

```diff
@@ -216,6 +216,9 @@ jobs:
         id: cmake_build
         run: |
           cmake -B build \
+            -DCMAKE_SYSTEM_NAME=Linux \
+            -DGGML_CCACHE=OFF \
+            -DGGML_NATIVE=OFF \
```

Comment:
I'm don't think I understand how this change fixed the CI for `ubuntu-cpu-make`?

Reply:
I was a bit extreme on this one and tried everything I could think of to make it work.

The tricky part is `CMAKE_SYSTEM_NAME=Linux`, which makes CMake believe we are cross-compiling (setting `CMAKE_CROSSCOMPILING`) but without breaking everything else. With that flag, i can disable `GGML_NATIVE_DEFAULT`:

https://github.com/ggml-org/llama.cpp/blob/835b2b915c52bcabcd688d025eacff9a07b65f52/ggml/CMakeLists.txt#L98-L103

then disabling `GGML_NATIVE` allows us not to set `INS_ENB`:

https://github.com/ggml-org/llama.cpp/blob/835b2b915c52bcabcd688d025eacff9a07b65f52/ggml/CMakeLists.txt#L134-L138

This way we get the lowest CPU mode. But honestly, `-DGGML_NATIVE=OFF` should be enough, and I also disabled `ccache` to increase my chances :)